dbt_platform_helper/providers/platform_config_schema.py

```diff
@@ -94,12 +92,12 @@ def __alb_schema() -> dict:
             Optional("environments"): {
                 PlatformConfigSchema.__valid_environment_name(): Or(
                     {
-                        Optional("additional_address_list"): list,
-                        Optional("allowed_methods"): list,
-                        Optional("cached_methods"): list,
+                        Optional("additional_address_list"): [str],
```

Comment:
I'm used to seeing this expressed as `list[str]`, e.g. https://docs.python.org/3/library/typing.html#type-aliases

What is prompting the `[str]` approach?

Reply:
list[str] is wrong. It allows a list of anything. It's type hint syntax not schema syntax.

It's not been noticed before as it is overly permissive.